programs/memo/src/memo.rs

```diff
@@ -0,0 +1,23 @@
+use {
+    mollusk_svm::Mollusk,
+    solana_sdk::{account::AccountSharedData, pubkey::Pubkey},
+};
+
+pub const ID: Pubkey = solana_sdk::pubkey!("MemoSq4gqABAXKb96qnH8TysNcWxMyWCqXgDLGmfcHr");
+
+pub fn add_program(mollusk: &mut Mollusk) {
+    mollusk.add_program_with_elf_and_loader(
+        &ID,
+        include_bytes!("elf/memo.so"),
```

Comment:
This macro adds a static slice, so we could optionally just make this a constant in the module, since it might be a little funky for devs trying to access the ELF again through the `target/debug` folder.